packages/common/src/entities/dbmss/dbmss.local.ts

```diff
@@ -131,11 +132,22 @@ export class LocalDbmss extends DbmssAbstract<LocalEnvironment> {
             throw new DbmsExistsError(`DBMS with name "${name}" already exists`);
         }
 
+        if (installPath) {
+            await fse.access(installPath, fse.constants.W_OK, (err) => {
+                throw new NotAllowedError(`Unable to create "${name}" in the location "${installPath}": ${err}`);
+            });
+        }
```

Comment:
If you pass a callback argument the function will not return a promise. 

```suggestion
            await fse.access(installPath, fse.constants.W_OK).catch((err) => {
                throw new NotAllowedError(`Unable to create "${name}" in the location "${installPath}": ${err}`);
            });
        }
```

Reply:
yeah Ive sorted it now. I was just pushing a change to make sure I could push to the correct branch